src/lib/bounded-contexts/marketing/marketing/tests/__tests__/send-email/send-email.steps.ts

```diff
@@ -1,16 +1,15 @@
-import { ContextBuilder } from '../../builders/context.builder';
```

Comment:
Maybe here also test the case that the emailService fails and returns an UnexpectedError for example like repos. This means that the interface of the Service will change to Either.